elytron-oidc-client/src/test/resources/org/wildfly/extension/elytron/oidc/elytron-oidc-client-preview-4.0.xml

```diff
@@ -157,6 +157,7 @@
         <logout-callback-path>/logout/myCallback</logout-callback-path>
         <post-logout-redirect-uri>http://localhost:8080/app/aCallback</post-logout-redirect-uri>
         <logout-session-required>true</logout-session-required>
+        <back-channel-logout-session-invalidation-limit>20</back-channel-logout-session-invalidation-limit>
```

Comment:
We should also make sure to update oidc-expressions.xml for each new option that has been added.

Reply:
I don't understand the format for this file
(e.g.  ${exp.truststore:truststore.jks} )
And doesn't the version "preview:3.0" need to be updated to "preview:4.0"

Reply:
done